src/components/container/CreateChallengeContainer.tsx

```diff
@@ -1,38 +1,98 @@
-import { useState } from "react";
-import { View } from "react-native";
+import { useState, useEffect } from "react";
+import { View, BackHandler, Alert } from "react-native";
 
+import { ChallengeStatusEnum } from "@src/interfaces/challenge.types";
 import ChallengeDetailsForm from "@src/components/forms/ChallengeDetailsForm";
 import ChallengeGoalForm from "@src/components/forms/ChallengeGoalForm";
 import ChallengeConfirmationForm from "@src/components/forms/ChallengeConfirmationForm";
 import ChallengeInvitationForm from "@src/components/forms/ChallengeInvitationForm";
 
+const initialValues = {
+  creator_id: "66693e1628ed06d3f5dcf64b", //TODO: Replace with useUser.getState().user.id when login/sign up is implemented
+  name: null,
+  description: null,
+  start_at: null,
+  end_at: null,
+  duration: 1,
+  goal: 80,
+  status: ChallengeStatusEnum.IN_PROGRESS,
+  members: [
```

Comment:
create challenge no longer need the creator_id and the members, is taken from the token